rust/core-lib/src/tabs.rs

```diff
@@ -95,6 +96,9 @@ pub const OPEN_FILE_EVENT_TOKEN: WatchToken = WatchToken(2);
 #[cfg(feature = "notify")]
 const THEME_FILE_EVENT_TOKEN: WatchToken = WatchToken(3);
 
+#[cfg(feature = "notify")]
+const PLUGIN_EVENT_TOKEN: WatchToken = WatchToken(4);
```

Comment:
Just something I was wondering, I'm not sure if it's ideal to manually set the "token id"? It might be better to generate the ID automatically when a new watch token is created and make sure it doesn't already exist. Well. I guess it's fine for now.

Reply:
I think the longer-term solution will be to have idle work sort of like the RPC system, where we have an enum of idle events that we can schedule and then that get passed in the callback. This implementation is really just an old C idiom, and yea it isn't perfect.